lib/netcore/Makefile.am

```diff
@@ -19,86 +19,25 @@
-			Thrift/SingletonTProcessorFactory.cs \
-			Thrift/TApplicationException.cs \
-			Thrift/TBaseClient.cs \
-			Thrift/TException.cs \
-			Thrift/TMultiplexedProcessor.cs \
-			Thrift/Collections/TCollections.cs \
-			Thrift/Collections/THashSet.cs \
-			Thrift/Properties/AssemblyInfo.cs \
-			Thrift/Protocols/ITProtocolFactory.cs \
-			Thrift/Protocols/TAbstractBase.cs \
-			Thrift/Protocols/TBase.cs \
-			Thrift/Protocols/TBinaryProtocol.cs \
-			Thrift/Protocols/TCompactProtocol.cs \
-			Thrift/Protocols/TJSONProtocol.cs \
-			Thrift/Protocols/TMultiplexedProtocol.cs \
-			Thrift/Protocols/TProtocol.cs \
-			Thrift/Protocols/TProtocolDecorator.cs \
-			Thrift/Protocols/TProtocolException.cs \
-			Thrift/Protocols/Entities/TField.cs \
-			Thrift/Protocols/Entities/TList.cs \
-			Thrift/Protocols/Entities/TMap.cs \
-			Thrift/Protocols/Entities/TMessage.cs \
-			Thrift/Protocols/Entities/TMessageType.cs \
-			Thrift/Protocols/Entities/TSet.cs \
-			Thrift/Protocols/Entities/TStruct.cs \
-			Thrift/Protocols/Entities/TType.cs \
-			Thrift/Protocols/Utilities/TBase64Utils.cs \
-			Thrift/Protocols/Utilities/TProtocolUtil.cs \
-			Thrift/Server/AsyncBaseServer.cs \
-			Thrift/Server/TBaseServer.cs \
-			Thrift/Server/TServerEventHandler.cs \
-			Thrift/Transports/TClientTransport.cs \
-			Thrift/Transports/TServerTransport.cs \
-			Thrift/Transports/TTransportException.cs \
-			Thrift/Transports/TTransportFactory.cs \
-			Thrift/Transports/Client/TBufferedClientTransport.cs \
-			Thrift/Transports/Client/TFramedClientTransport.cs \
-			Thrift/Transports/Client/THttpClientTransport.cs \
-			Thrift/Transports/Client/TMemoryBufferClientTransport.cs \
-			Thrift/Transports/Client/TNamedPipeClientTransport.cs \
-			Thrift/Transports/Client/TSocketClientTransport.cs \
-			Thrift/Transports/Client/TStreamClientTransport.cs \
-			Thrift/Transports/Client/TTlsSocketClientTransport.cs \
-			Thrift/Transports/Server/THttpServerTransport.cs \
-			Thrift/Transports/Server/TNamedPipeServerTransport.cs \
-			Thrift/Transports/Server/TServerFramedTransport.cs \
-			Thrift/Transports/Server/TServerSocketTransport.cs \
-			Thrift/Transports/Server/TTlsServerSocketTransport.cs 
+			Thrift/Thrift.csproj
 
 all-local: \
 	Thrift.dll
 
 Thrift.dll: $(THRIFTCODE)
-	$(MKDIR_P) $(GENDIR)
-	$(THRIFT)  -gen netcore:wcf   -r  -out $(GENDIR)  $(TESTDIR)/CassandraTest.thrift
-	$(THRIFT)  -gen netcore:wcf   -r  -out $(GENDIR)  $(top_srcdir)/test/ThriftTest.thrift
-	$(THRIFT)  -gen netcore:wcf   -r  -out $(GENDIR)  $(top_srcdir)/contrib/fb303/if/fb303.thrift
-	$(DOTNETCORE) --info
-	$(DOTNETCORE) restore
+#	$(MKDIR_P) $(GENDIR)
+#	$(THRIFT)  -gen netcore:wcf   -r  -out $(GENDIR)  $(TESTDIR)/CassandraTest.thrift
+#	$(THRIFT)  -gen netcore:wcf   -r  -out $(GENDIR)  $(top_srcdir)/test/ThriftTest.thrift
+#	$(THRIFT)  -gen netcore:wcf   -r  -out $(GENDIR)  $(top_srcdir)/contrib/fb303/if/fb303.thrift
 	$(DOTNETCORE) build
 
 clean-local:
-	$(RM) Thrift.dll
```

Comment:
Why aren't we cleaning up Thrift.dll as part of clean?

Reply:
Nice comment - maybe we should remove auto-generated files